src/client.cpp

```diff
@@ -1361,8 +1361,22 @@ void Client::removeNode(v3s16 p)
 	}
 }
 
+/**
+ * Helper function for Client Side Modding
+ * Flavour is applied there, this should not be used for core engine
+ * @param p
+ * @param is_valid_position
+ * @return
+ */
 MapNode Client::getNode(v3s16 p, bool *is_valid_position)
```

Comment:
it has as right place as privileges :), and not we cannot specialize a function to a unrelated object feature